app/helpers/application_helper/button/basic.rb

```diff
@@ -12,6 +12,11 @@ def initialize(view_context, view_binding, instance_data, props)
     end
   end
 
+  def check_rbac?
+    return true if self[:child_id].nil?
+    role_allows?(:feature => self[:child_id])
```

Comment:
Yeah, I have no idea what a child_id means in terms of ProductFeature nor a Button.

Yes, multiple entitlements isn't done yet, just a heads-up that how you call `role_allows?` will be changing in the future.